### PR TITLE
Feat/aggregate time to art report tables

### PR DESCRIPTION
**Pull request by nthusi-codes**
This feature populates the time to art aggregate report tables

### CODE REVIEW

**Comment by nobert-mumo** [comment]
@nthusi-codes good stuff with moving this to the new design! Just checking: Did you avoid having the 3 tables combined into one intentionally?

**Comment by nobert-mumo** [comment]
@nthusi-codes for the time groupings maybe we can already have the 'yyyy-mm' column as opposed to separate year & month columns? I think it makes it easier for a user when building visualizations.

**Review by Marymary-dev — APPROVED** [review]

**Review by cbrianbet — APPROVED** [review]

**Comment by nobert-mumo** [comment]
> > @nthusi-codes good stuff with moving this to the new design! Just checking: Did you avoid having the 3 tables combined into one intentionally?
> 
> Yes. Realized that when working on the ART theme, the aggregations are either specific to their underlying 'where' clause or group clause which aren't the same for all tables. eg, last 12m aggregates based on the last one year's records only. Merging this with any other table may not give the intended results. With the other themes, I've found tables that can be merged though

Ah okay, makes sense, ideally if it is more easily understandable as a standalone table, we should leave them as separate tables.

File: Scripts/REPORTING/load_aggregate_time_to_art_grp.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+MFLCode,
+f.FacilityName,
+County,
+Subcounty,
+p.PartnerName as CTPartner,
+a.AgencyName as CTAgency,
+Gender,
+g.DATIMAgeGroup as AgeGroup,
+Year(StartARTDateKey) StartARTYear,
+DateName(Month,StartARTDateKey) StartARTMonth,
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_time_to_art_grp.sql:17]: @nthusi-codes maybe for the Year and Month we can combine them into one column ?

Comment (nthusi-codes) [reply Scripts/REPORTING/load_aggregate_time_to_art_grp.sql:17]: This means it having it as a 'yyyy-mm' varchar data type right. Will we be able to filter using, let's say year and not month, if we did this?

Comment (nthusi-codes) [reply Scripts/REPORTING/load_aggregate_time_to_art_grp.sql:17]: I've added the new column but also left the separate year and month column for now. I'd like to see how the filtering would work in a BI tool then I'll remove the two redundant columns

Comment (nobert-mumo) [reply Scripts/REPORTING/load_aggregate_time_to_art_grp.sql:17]: @nthusi-codes sure enough, indeed might be a good way to test the user experience for filtering. My guess is most probably a user would want to filter/group on a year-month combination.

**Review by nobert-mumo — APPROVED** [review]

**Pull request merged**